packages/app/src/systems/Account/services/account.ts

```diff
@@ -149,7 +156,19 @@ export class AccountService {
     // TODO: fix this on fuel-ts it should be possible to
     // customize the ProviderURL on the manager level
     wallet.provider = new Provider(VITE_FUEL_PROVIDER_URL);
-    return wallet;
+
+    return {
+      ...(wallet as WalletUnlocked),
+      exportVault() {
```

Comment:
I think we should implement this in another way to strictly expose mnemonic only when it's needed.

Maybe passing some configs when spawning the unlock machine.